src/Raven.Client/Documents/Operations/TimeSeries/TimeSeriesOperation.cs

```diff
@@ -133,6 +171,16 @@ internal static TimeSeriesOperation ParseForBulkInsert(BlittableJsonReaderObject
             result.Appends = new List<AppendOperation>(sorted.Values);
 
             return result;
+
+            static long GetLong(object value)
```

Comment:
Will this box? should we `Inline` this?

Reply:
No, this shouldn't cause boxing. Why do you expect it to?